py/setup.py

```diff
@@ -6,7 +6,7 @@
 here = path.abspath(path.dirname(__file__))
 
 # Get the long description from the relevant file
-with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
+with open(path.join(here, 'README_a.rst'), encoding='utf-8') as f:
```

Comment:
Why it's changed?

Reply:
Debugging change leaked in, should be `README`, thx

Reply:
Fixed